cli/kaos_cli/commands/build.py

```diff
@@ -27,7 +32,7 @@ def build():
 
 @build.command(name='deploy',
                short_help='{}'.format(
-                   click.style('Build the kaos backend', bold=True, fg='black')))
+                   click.style('Build the kaos backend', bold=False, fg='white')))
```

Comment:
Maybe even remove the `fg='white'` argument? The others don't have it, and I think it's better left for the shell to interpret the values based on the theme.

Reply:
Yes, this can be left unspecified. Change included